### DISCUSSION TITLE
math.h in zephyr

### DISCUSSION THREAD

**Discussion by JasmineRandhawa** [zephyrproject-rtos/zephyr#29988] [Q&A] [answered]
How to I calll pow() function for float exponential ?

**Comment by ejohnso49** [comment, marked as answer]
To my knowledge all you need to do is add `CONFIG_NEWLIB_LIBC=y` to your prj.conf file. This will allow you to call functions within `math.h`

**Reply by JasmineRandhawa** [reply]
Thanks so much @ejohnso49 :)
